k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: rpe-service-auth-provider
+  namespace: software-engineering
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.podinfod: "glob:latest*"
+spec:
+  releaseName: rpe-service-auth-provider
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo
+    name: rpe-service-auth-provider
+    version: 0.0.1
+  values:
+    java:
+      applicationPort: 8489
+      keyVaults:
+        "s2s":
```

Comment:
isn't this already in the chart?

Reply:
They are in values.template.yaml . I am assuming they won't be picked?

Reply:
they will be